### DISCUSSION TITLE
SSL certification problems

### DISCUSSION THREAD

**Discussion by audiohustla** [Guilouz/Creality-Helper-Script-Wiki#523] [Issues]
After upgrading my K1 max to version 1.3.3.8 and resetting factory settings, I cant intall moonraker or anything else, it always reports with some ssl certification problems, Ive already rebooted the printer twice, nothing seems to work. Any suggestions?

**Comment by TheWebMachine** [comment]
From the Wiki:

- If you encounter an issue to clone Helper Script repository, enter this command before cloning:

SSH Command Prompt
`git config --global http.sslVerify false`

- Also, make sure your system date and time are correct. See Change Date & Time section.

**Reply by audiohustla** [reply]
I tried that before, I can install the helper script itself, but afterwards I cant install anything. Also the Date changes wont apply correct, if I choose Europe/Berlin it still shows the wrong day/time only the right timezone

**Comment by audiohustla** [comment]
So After going back to 1.3.3.5 it works again, it also accepts the right timezone, for now I will stay with this Version

**Comment by pauloeduardogodoy** [comment]
Are you using Windows Command Prompt to SSH the printer?

**Comment by The-last-phantom** [comment]
I get the same issue with the date and time. The timezone updates correctly when using the guide, but the actual date and time does not change. Mine currently still says it is Monday 9 March 2020. Using MobaXterm to SSH the printer.

*EDIT* I've also just rolled back to V1.3.3.5 and can confirm it all works now as @audiohustla experienced as well, so there must be an issue with the way the date/time script makes changes in V1.3.3.8?

**Comment by igorcstab** [comment]
I had the same issue as you but someone suggested a thing that worked for me: I've manually set the date/time with
`date -s "2024-07-27 17:45:00"`
It looks like the ntp isn't working properly on my FW version, which is 1.3.3.29 but a manual set worked to make the SSL checks stop complaining. So there's no need, apparently to rollback to an older FW version
